RadixWallet/Features/TransactionReviewFeature/TransactionReview+Sections.swift

```diff
@@ -99,7 +98,7 @@ extension TransactionReview {
 			let allAddresses = allAddresses + resourceAddresses.asIdentifiable()
 			let resourcesInfo = try await resourcesInfo(allAddresses.elements)
 
-			let dApps = await extractDappEntities(poolAddresses)
+			let dApps = await extractDappEntities(poolAddresses.map(\.asGeneral))
```

Comment:
will need to upgrade code in TX review to make use of the specific address, and not downcast to `Address`....